microprofile-jwt/src/test/java/org/jboss/eap/qe/microprofile/jwt/security/rbac/RolesAllowedRbacTest.java

```diff
@@ -222,7 +223,7 @@ public void nonAuthenticatedAccessAdminDirectorPath(@ArquillianResource URL url)
      */
     @Test
     public void nonAuthenticatedAccessDenyAllPath(@ArquillianResource URL url) {
-        get(url.toExternalForm() + "rbac-endpoint/deny-all")
+        get(url.toExternalForm() + Endpoints.RBAC_ENDPOINT + "/deny-all")
```

Comment:
Any reason why not move `deny-all` to a constant?

Reply:
Good point! Thanks for finding this. Will update.